pages/index.js

```diff
@@ -1,40 +1,85 @@
-import Heading from "@/Header";
 import InterviewCard from "@/components/InterviewCard";
 import { interviews } from "@/public/interviews";
 import SearchInput from "@/components/Search/SearchInput";
 import SearchResults from "@/components/Search/SearchResults";
 import { useState } from "react";
 import Footer from "@/components/Footer";
+import styled from "styled-components";
 
+const INTERVIEWS_PER_PAGE = 2;
 export default function HomePage() {
   const [searchTerm, setSearchTerm] = useState("");
-
+  const [currentPage, setCurrentPage] = useState(1);
+  const totalPages = Math.ceil(interviews.length / INTERVIEWS_PER_PAGE);
+  const paginatedInterviews = interviews.slice(
+    (currentPage - 1) * INTERVIEWS_PER_PAGE,
+    currentPage * INTERVIEWS_PER_PAGE
+  );
   function handleSearch(term) {
     setSearchTerm(term);
   }
   return (
     <>
-      <main>
+      <StyledMain>
         <SearchInput searchTerm={searchTerm} onSearch={handleSearch} />
         {searchTerm ? (
           <SearchResults searchTerm={searchTerm} />
         ) : (
-          <ul>
-            {interviews.map((interview) => (
-              <li className="interview-card" key={interview.id}>
-                <InterviewCard
-                  id={interview.id}
-                  title={interview.title}
-                  summary={interview.summary}
-                  text={interview.text}
-                  image={interview.image}
-                />
-              </li>
-            ))}
-          </ul>
+          <>
+            <ul>
+              {paginatedInterviews.map((interview) => (
+                <li className="interview-card" key={interview.id}>
+                  <InterviewCard
+                    id={interview.id}
+                    title={interview.title}
+                    summary={interview.summary}
+                    text={interview.text}
+                    image={interview.image}
+                  />
+                </li>
+              ))}
+            </ul>
+
+            {currentPage === 1 && (
+              <PageNavBtn onClick={() => setCurrentPage(2)}>
+                Next Page
+              </PageNavBtn>
+            )}
+            {currentPage === 2 && (
+              <>
+                <PageNavBtn onClick={() => setCurrentPage(1)}>
+                  Previous Page
+                </PageNavBtn>
+                <PageNavBtn onClick={() => setCurrentPage(3)}>
+                  Next Page
+                </PageNavBtn>
+              </>
+            )}
+            {currentPage === 3 && (
+              <PageNavBtn onClick={() => setCurrentPage(2)}>
+                Previous Page
+              </PageNavBtn>
+            )}
```

Comment:
I think this will probably work based on the current number of Interviews that you have. But would it still work if you had 50 interviews?

We might want to consider making it more dynamic. 

Maybe something like

``` js
{/* Show next button if there are more pages (i.e. currentPage is less than the total number of pages) */}
{currentPage < totalPages && (
  <PageNavBtn onClick={() => setCurrentPage(currentPage + 1)}>
    Next Page
  </PageNavBtn>
)}
{/* Show previous button if we are on any page greater than 1 */}
{currentPage > 1 && (
  <PageNavBtn onClick={() => setCurrentPage(currentPage - 1)}>
    Previous Page
  </PageNavBtn>
)}

```

Reply:
Thank you! That makes a lot of sense. Should have thought about this...